src/pages/en.md

```diff
@@ -316,6 +316,10 @@ For example, Rails has a [`rails-dev-box`](https://github.com/rails/rails-dev-bo
 
 [Docker](https://www.docker.com) is also a good way to make dependency management simpler.
 
+- [ ] Promptly update version of gem at RubyGems
```

Comment:
Yes, I actually meant that you must not to forget to update RubyGems' version of gem when you update your gem version in repository. And conditions for making a decision are what we really need.

We can create separate section "Publishing" for this point and point about RC (as individual one).